### ISSUE TITLE
Cookie expires in a very short term

### ISSUE THREAD

**Issue by nlpsuge** [sunwxg/gnome-shell-extension-dict#9]
Cookie expires in a very short term after logging in a site within this extension.

But if I log in using a regular browser such as Chrome, I'll not be logged out unless I delete the cookies manually.

**Comment by sunwxg** [comment]
Cookie is not saved in this extension.

**Comment by nlpsuge** [comment]
Will you support this feature to save cookies?

It should be a big time-saver, because I'm able to save the result(s) by using the online service such as dictionary.cambridge.org, etc.

**Comment by sunwxg** [comment]
Okay, I will consider to add a option to enable cookie.

**Comment by nlpsuge** [comment]
Hi,

I've updated to the latest version, which support to store cookie. And I saw your code too. 

But I couldn't find this folder '/run/dict' was created. Is this normal?

**Comment by sunwxg** [comment]
Yes, I also can't find '/run/dict', I don't know the reason now.
`context.get_cookie_manager().set_accept_policy(Webkit.CookieAcceptPolicy.ALWAYS);`
This line enable the cookie.

**Comment by nlpsuge** [comment]
Try to change the name '/run/dict' to 'dict', I could find a folder named 'dict' in the home folder. Not true about if it's helpful for persisting the cookies. By the way, it seems that the cookies will be expired after reboot.
